test/infer/test_svi.py

```diff
@@ -85,11 +193,12 @@ def body_fn(i, val):
         svi_state, _ = svi.update(val, data)
         return svi_state
 
-    svi_state = fori_loop(0, 2000, body_fn, svi_state)
+    svi_state = fori_loop(0, 4000, body_fn, svi_state)
     params = svi.get_params(svi_state)
+    actual_posterior_mean = 0.75  # (8 + 1) / (8 + 1 + 2 + 1)
     assert_allclose(
         params["alpha_q"] / (params["alpha_q"] + params["beta_q"]),
-        0.8,
+        actual_posterior_mean,
```

Comment:
does this pass with tighter tolerance?

Reply:
it passes with a bit smaller tolerance and tuning

Reply:
I think it's tricky to use small tolerance for this test (both elbo and renyi-elbo): the losses converge and fluctuate around the optimal value.